manifests/crc/BYODB-installation/openshift/kruize-crc-openshift.yaml

```diff
@@ -35,10 +35,10 @@ data:
       "hibernate": {
         "dialect": "org.hibernate.dialect.PostgreSQLDialect",
         "driver": "org.postgresql.Driver",
-        "c3p0minsize": 2,
-        "c3p0maxsize": 5,
+        "c3p0minsize": 10,
```

Comment:
Please set it to following values    As i was getting too many client request error
        
        "c3p0minsize": 5,
        "c3p0maxsize": 10,
        "c3p0timeout": 300,
        "c3p0maxstatements": 100,
        "hbm2ddlauto": "none",